modules/dcache-frontend/src/main/java/org/dcache/restful/resources/migration/MigrationResources.java

```diff
@@ -232,9 +242,108 @@ public Response submitMigrationCopy(@ApiParam(
         } catch (InterruptedException | CacheException | NoRouteToCellException e) {
             throw new InternalServerErrorException(e);
         }
+    }
+
+    /**
+     * Gets migration information to a specified migration ID on a pool.
+     */
+    @GET
+    @ApiOperation(value = "Gets information to a migration. (See Pool Operator Commands 'migration info')")
```

Comment:
Maybe  "information to a" -> "information for a"

Reply:
Agreed. Fixed in [#e0a1266](https://github.com/dCache/dcache/pull/7040/commits/e0a1266c9d7b00f1699821fa31979c19c6f89668)